.github/workflows/pr-environment.yml

```
@@ -0,0 +1,50 @@
+on:  
+  pull_request:
+    types: [opened, synchronize, reopened, closed]
+
+jobs:
+  deploy:
+    if: ${{ github.event_name == 'pull_request' && contains(github.event.pull_request.title, '[no-depl]') == false && github.event.action != 'closed' }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)